stingray/simulator/fake.py

```diff
@@ -89,7 +89,7 @@ def fake_events_from_lc(
             rough_bins = np.rint((random_ts - t0) / bin_time)
             rough_bins = rough_bins.astype(int)
 
-            pts = lc_filt[rough_bins]
+            pts = [lc_filt[bin] for bin in rough_bins]
```

Comment:
Why did you change this? This filter is going to run on numpy arrays, so it will work as it was.